x-pack/test/alerting_api_integration/common/lib/object_remover.ts

```diff
@@ -27,7 +27,7 @@ export class ObjectRemover {
     spaceId: ObjectToRemove['spaceId'],
     id: ObjectToRemove['id'],
     type: ObjectToRemove['type'],
-    plugin: ObjectToRemove['plugin'],
+    plugin?: ObjectToRemove['plugin'],
```

Comment:
I think If we change the base bath of the maintenance window, we will not need these changes.